### PR TITLE
⏩ #1230 Migration script for messages 

### PR DESCRIPTION
**Pull request by stephanevieira75**

### CODE REVIEW

File: twake/backend/node/src/services/messages/api.ts [hunk 1]
```diff
@@ -12,15 +12,21 @@ import {
...
   ThreadExecutionContext,
   MessageWithReplies,
 } from "./types";
+
 import { ParticipantObject, Thread, ThreadPrimaryKey } from "./entities/threads";
 import { Message, MessagePrimaryKey } from "./entities/messages";
+import {
+  PhpMessage,
+  PhpMessagePrimaryKey,
+} from "../../cli/cmds/migration_cmds/php-message/php-message-entity";
```
Comment (chamerling) [inline twake/backend/node/src/services/messages/api.ts:29]: This is something that MUST not happen. The runtime depends on some CLI code which is bad

Comment (chamerling) [reply twake/backend/node/src/services/messages/api.ts:29]: A solution will be to have a `legacy` folder where both CLI and the message API depends on

File: twake/backend/node/src/services/console/processing/merge.ts [hunk 2]
```diff
@@ -54,7 +55,7 @@ export class MergeProcess {
   ) {
     this.client = new ConsoleHTTPClient(consoleClientParameters, dryRun);
   }
-
+  //
```
Comment (RomaricMourgues) [inline twake/backend/node/src/services/console/processing/merge.ts:58]: Why modifying this code ?

File: twake/backend/node/src/services/messages/entities/messages.ts [hunk 3]
```diff
@@ -88,7 +88,7 @@ export class Message {
 
 export type MessageReaction = { count: number; name: string; users: string[] };
 
-export type MessageOverride = { title?: string; picture?: string };
+export type MessageOverride = { title?: string; picture?: string; type?: string };
```
Comment (RomaricMourgues) [inline twake/backend/node/src/services/messages/entities/messages.ts:91]: What is type here?

**Pull request merged**